bin/s2nc.c

```diff
@@ -238,6 +245,31 @@ int main(int argc, char *const *argv)
 
     struct s2n_connection *conn = s2n_connection_new(S2N_CLIENT);
 
+    if (mfl_value > 0) {
+        switch(mfl_value) {
+            case 512:
+                mfl_code = S2N_TLS_MAX_FRAG_LEN_512;
+                break;
+            case 1024:
+                mfl_code = S2N_TLS_MAX_FRAG_LEN_1024;
```

Comment:
Since these come from the command line I think `512` is more ergonomic than `S2N_TLS_MAX_FRAG_LEN_512` :)